src/main/java/com/amazon/opendistroforelasticsearch/security/resolver/IndexResolverReplacer.java

```diff
@@ -172,7 +155,8 @@ private Resolved resolveIndexPatterns(final IndicesOptions indicesOptions, final
         }
 
         Set<String> remoteIndices;
-        final List<String> localRequestedPatterns = new ArrayList<>(Arrays.asList(requestedPatterns0));
+        final List<String> localRequestedPatterns = Lists.newArrayList(requestedPatterns0);
```

Comment:
`Lists.newArrayList()` is likely to be deprecated in favor of the original code.